assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -22,23 +23,120 @@ function MultiSelectionMovable( { selectedElements, nodesById } ) {
 		if ( moveable.current ) {
 			moveable.current.updateRect();
 		}
-	} );
+	}, [ selectedElements, moveable, nodesById ] );
+
+	const { actions: { updateElementsById } } = useStory();
 
-	const targets = selectedElements.map( ( element ) => nodesById[ element.id ] );
+	// Create targets list including nodes and also necessary attributes.
+	const targetList = selectedElements.map( ( element ) => {
+		return {
+			ref: nodesById[ element.id ],
```

Comment:
The `ref` naming here is slightly confusing since it usually indicates `Ref` type. Instead we have a `node`. Maybe rename to `node`?

Reply:
Good point, changed.